new-dti-website/src/app/layout.tsx

```diff
@@ -11,8 +11,10 @@ export const ibm_plex_mono = IBM_Plex_Mono({
 
 const RootLayout = ({ children }: { children: React.ReactNode }): JSX.Element => (
   <html lang="en">
-    <body className={`${inter.className} bg-black relative overflow-x-hidden`}>
-      <Page>{children}</Page>
+    <body className={`${inter.className} bg-black`}>
```

Comment:
Same thing here when we're passing in the dynamic font since nextjs/fonts calls are different. I'm thinking we should maybe use the same method of either using the global css or the tailwind config and stick to it!